src/StripeTests/Entities/ExternalAccounts/ExternalAccountTest.cs

```diff
@@ -8,17 +8,17 @@ namespace StripeTests
     using Stripe;
     using Xunit;
 
-    public class ExternalAccountTest : BaseStripeTest
+    public class IExternalAccountTest : BaseStripeTest
     {
         [Fact]
         public void Deserialize()
         {
             string json = GetFixture("/v1/accounts/acct_123/external_accounts/ba_123");
-            var externalAccount = Mapper<ExternalAccount>.MapFromJson(json);
+            var externalAccount = Mapper<IExternalAccount>.MapFromJson(json);
             Assert.NotNull(externalAccount);
-            Assert.Equal(ExternalAccountType.BankAccount, externalAccount.Type);
-            Assert.NotNull(externalAccount.BankAccount);
-            Assert.Equal("bank_account", externalAccount.BankAccount.Object);
+            Assert.IsType<BankAccount>(externalAccount);
+            var bankAccount = (BankAccount)externalAccount;
+            Assert.Equal("bank_account", bankAccount.Object);
```

Comment:
Same you mentioned you did not need to use `var` anymore

Reply:
I forgot to update this test. But as said above, using `dynamic` is optional, it's basically syntactic sugar at the cost of type safety.

Reply:
Thinking about this a bit more, I think we could make `Object` part of the `IStripeEntity` interface (or maybe on a new `IStripeEntityWithObject` interface) so that we can check its value without using `dynamic`.